lms/djangoapps/support/static/support/jsx/entitlements/components/Main/MainContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+
+import Main from './Main';
```

Comment:
If you're importing `Main` this way, then shouldn't it's file path be `/Main/index.jsx` not `/Main/Main.jsx`?

Reply:
currently my files are ```/Main/Main.jsx and /Main/MainContainer.jsx``` I've considered changing those to index and container, (for other components as well) but haven't decided my preference yet, do we have a style guide/decision on that? @arizzitano @edx/fedx-team